pkg/detector/ospkg/rootio/rootio.go

```diff
@@ -73,7 +77,7 @@ func (s *Scanner) Detect(ctx context.Context, osVer string, _ *ftypes.Repository
 			srcName = pkg.Name
 		}
 
-		advisories, err := s.vsg.Get(osVer, srcName)
+		advisories, err := s.vsg.Get(osVer, srcName, pkg.Arch)
```

Comment:
We merged https://github.com/aquasecurity/trivy/pull/9239 with updated Getter interface.

Reply:
Use GetParams here